Author: Radek Polak <psonek2@seznam.cz>
Subject: dump SMS timestamp
Origin: upstream, http://git.freesmartphone.org/?p=cornucopia.git;a=commit;h=5530cf7f4d51ad1c6b59fd1b7a59e85d67478bd5
Bug-Debian: http://bugs.debian.org/643100

diff --git a/src/lib/fsogsm3rdparty.vapi b/src/lib/fsogsm3rdparty.vapi
index 338b96a..23e2e27 100644
--- a/src/lib/fsogsm3rdparty.vapi
+++ b/src/lib/fsogsm3rdparty.vapi
@@ -359,6 +359,7 @@ namespace Sms
             props.insert( "pid", pid );
             props.insert( "dcs", dcs );
             props.insert( "udl", udl );
+            props.insert( "timestamp", scts.to_string() );
             if ( udhi )
             {
                 props.insert( "udh", "%02X %02X %02X %04X %04X".printf( ud[0], ud[1], ud[2], ud[4] + (ud[3] << 8), ud[6] + (ud[5] << 8) ) );
